packages/components/src/components/form/Textarea/Textarea.stories.tsx

```diff
@@ -1,32 +1,19 @@
-import { ChangeEvent } from 'react';
-import { useArgs } from '@storybook/client-api';
-
 import { Textarea as TextareaComponent } from './Textarea';
 
 export default {
     title: 'Form/Textarea',
+    component: TextareaComponent,
+};
+
+export const Textarea = {
     args: {
-        value: 'Textarea',
+        defaultValue: 'Textarea',
         label: 'Label',
-        bottomText: '',
-        placeholder: '',
-        disabled: false,
-        isMonospace: false,
-        state: null,
         rows: 5,
+        maxLength: 500,
+        characterCount: true,
```

Comment:
doesn't work in Storybook :(

Reply:
Fixed in 4e9b4d1d39b2924184b3aade5fa8fb482a0a4540